### PR TITLE
Added a callout about -m flag

### LINKED ISSUE #163: explain -m flag
Maybe we should add a callout that explains the `-m` flag in the `python -m venv venv` command in the [section 1 virtual environments episode](https://carpentries-incubator.github.io/python-intermediate-development/12-virtual-environments/index.html)? - answer running the venv module as a script

### PR DESCRIPTION
**Pull request by anenadic**
Fixes #163.

### CODE REVIEW

File: _episodes/12-virtual-environments.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ keypoints:
 - "`pip` allows you to declare all dependencies for a project in a separate
 file (by convention called `requirements.txt`) which can be shared with collaborators/users and used to replicate a virtual environment."
 - "Use `pip3 freeze > requirements.txt` to take snapshot of your project's dependencies."
-- "Use `pip3 install -r requirements.txt` to replicate someone else's virtual environment on your machine from
+- "Use `pip3 install -r requirements.txt` to replicatefpython3 -m someone else's virtual environment on your machine from
```
Comment (steve-crouch) [inline _episodes/12-virtual-environments.md]: Probably needs correction :)

Comment (anenadic) [inline _episodes/12-virtual-environments.md]:
```suggestion
- "Use `pip3 install -r requirements.txt` to replicate someone else's virtual environment on your machine from
```

File: _episodes/12-virtual-environments.md [hunk 2]
```diff
@@ -220,6 +220,18 @@ conventionally within your software project so they are co-located.
...
 (and any parent directories that don’t exist already).
 
+> ## What is `-m` Flag in `python3` Command??
+> The Python `-m` flag means "module" and tells the Python interpreter to treat what follows `-m` 
+> as the name of a module and not as a single, executable program with the same name. Some modules 
+> (such as `venv` or `pip`) have main entry points and the `-m` flag can be used to invoke them 
+> on the command line via the `python` command. The main difference between running such modules as 
+> standalone programs (e.g. executing "venv" by running the `venv` command directly) 
+> versus using `python3 -m` command seems to be that with latter you are in full control of 
+> which `venv` module will be invoked (the one that came with your Python interpreter vs.
```
Comment (anenadic) [inline _episodes/12-virtual-environments.md:230]:
```suggestion
> which Python module will be invoked (the one that came with your Python interpreter vs. 
```

Comment (steve-crouch) [inline _episodes/12-virtual-environments.md]:
```suggestion
> which Python module will be invoked (the one that came with your Python interpreter vs. 
```

**Review by steve-crouch — CHANGES_REQUESTED** [review]

File: _episodes/12-virtual-environments.md [hunk 3]
```diff
@@ -220,6 +220,18 @@ conventionally within your software project so they are co-located.
...
 (and any parent directories that don’t exist already).
 
+> ## What is `-m` Flag in `python3` Command??
+> The Python `-m` flag means "module" and tells the Python interpreter to treat what follows `-m` 
+> as the name of a module and not as a single, executable program with the same name. Some modules 
+> (such as `venv` or `pip`) have main entry points and the `-m` flag can be used to invoke them 
+> on the command line via the `python` command. The main difference between running such modules as 
+> standalone programs (e.g. executing "venv" by running the `venv` command directly) 
+> versus using `python3 -m` command seems to be that with latter you are in full control of 
+> which Python module will be invoked (the one that came with your Python interpreter vs.
```
Comment (anenadic) [inline _episodes/12-virtual-environments.md:230]:
```suggestion
> which Python module will be invoked (the one that came with your environment's Python interpreter vs. 
```

File: _episodes/12-virtual-environments.md (hunk 1 above)
Comment (anenadic) [reply _episodes/12-virtual-environments.md]: fixed via to our virtual environment

File: _episodes/12-virtual-environments.md (hunk 2 above)
Comment (anenadic) [reply _episodes/12-virtual-environments.md]: fixed via 7ee460d2025cdacf3223f985758c08408588e3f9

**Review by steve-crouch — APPROVED** [review]

**Pull request merged**